### PR TITLE
IsNotEmpty

### PR DESCRIPTION
**Pull request by AWilliamson88**
need your honest opinion.

### CODE REVIEW

File: Validation_Tests/ValidationsTests/IsNotEmptyValidationTests/IsNotEmptyValidationTests.cs [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+using FluentAssertions.Execution;
+using PopValidations.Validations;
+using PopValidations_Tests.ValidationsTests.GenericTestableObjects;
+using System;
+using System.Collections.Generic;
+using Xunit;
+
+namespace PopValidations_Tests.ValidationsTests;
+
+public class IsNotEmptyValidationTests
```
Comment (AdamWilliamson) [inline Validation_Tests/ValidationsTests/IsNotEmptyValidationTests/IsNotEmptyValidationTests.cs]: You should be getting namespace conflicts., Im guessing you did, and deleted the namespce.
If you checked the other files, you'd have seen the solution to this.
There should be an _  between Validation and Tests. in this class/file name.
And the namespace needs to be updated to the folder name.

File: Validation_Tests/ValidationsTests/IsNotEmptyValidationTests/IsNotEmptyValidation_RunnerTests.GivenAValidator_WithErrors_ThenEveryFieldHasErrors.approved.json [hunk 2]
```diff
@@ -0,0 +1,160 @@
+﻿{
```
Comment (AdamWilliamson) [inline Validation_Tests/ValidationsTests/IsNotEmptyValidationTests/IsNotEmptyValidation_RunnerTests.GivenAValidator_WithErrors_ThenEveryFieldHasErrors.approved.json:1]: This is no longer the correct format, for this file.

You should consider doing a rebase on master.
And re-running all the tests.

**Review by AdamWilliamson — CHANGES_REQUESTED** [review]

File: Validation_Tests/ValidationsTests/IsNotEmptyValidationTests/IsNotEmptyValidationTests.cs (hunk 1 above)
Comment (AWilliamson88) [reply Validation_Tests/ValidationsTests/IsNotEmptyValidationTests/IsNotEmptyValidationTests.cs]: My bad, I hadn't fetched the repo in far too long it seems.
I only fetched the latest version just before submitting the pull request.
It said there weren't any merge conflicts and I didn't consider that there'd be so many changes.
I should have run the tests again... my bad.

File: Validation_Tests/ValidationsTests/IsNotEmptyValidationTests/IsNotEmptyValidation_RunnerTests.GivenAValidator_WithErrors_ThenEveryFieldHasErrors.approved.json (hunk 2 above)
Comment (AWilliamson88) [reply Validation_Tests/ValidationsTests/IsNotEmptyValidationTests/IsNotEmptyValidation_RunnerTests.GivenAValidator_WithErrors_ThenEveryFieldHasErrors.approved.json:1]: Why a rebase?

**Review by AdamWilliamson — APPROVED** [review]

**Pull request merged**